### PR TITLE
snow microstructure tutorial placeholder

### PR DESCRIPTION
**Pull request by mikedurand**
I created a placeholder for the microstructure tutorial. just trying out the workflow, mostly.

### CODE REVIEW

File: book/_toc.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@
           sections:
             - file: tutorials/sar/sentinel1
             - file: tutorials/sar/uavsar
+            - file: tutorials/microstructure/microstructure-tutorial
```
Comment (scottyhq) [inline book/_toc.yml:34]: this is currently under the SAR tutorial section, so you'll want to move it up one level:
```yaml
- part: Tutorials
  chapters:
    - file: tutorials/index
      sections:
        - file: tutorials/microstructure/microstructure-tutorial
```

Comment (mikedurand) [reply book/_toc.yml]: done! I think. :) moved the tutorial to its own section.

File: book/tutorials/microstructure/microstructure-tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Microstructure Tutorial\n",
+    "\n",
+    "Snow microstructure is important - it governs both visible and radar remote sensing to snow cover. \n",
+    "\n",
+    "Snow microstructure is controlled by other snow properties, such as snow temperature, snow height and snow liquid water content. \n",
+    "\n",
+    "# SnowEx Microstructure Measurement Background\n",
```
Comment (scottyhq) [inline book/tutorials/microstructure/microstructure-tutorial.ipynb:13]: JupyterBook parses markdown headers for website navigation. You can see how this is rendered on the preview link in this pull_request:  https://deploy-preview-34--snowex-hackweek.netlify.app/tutorials/microstructure/microstructure-tutorial.html . Long story short, you want just a single header (#Microstructure Tutorial, and then subsquent subheaders for example:
```
# Microstructure Tutorial
## Background
### Background subsection
## Data
```

Comment (mikedurand) [reply book/tutorials/microstructure/microstructure-tutorial.ipynb]: got it! just pushed a fix.

**Comment by scottyhq** [comment]
looking good @mikedurand . don't worry about the failing 'quality control' test (https://github.com/snowex-hackweek/website/pull/34/checks?check_run_id=2746873003) , it's just a finicky hyperlink to an image in the sar tutorial. we can either merge this as is or you can keep pushing commits with changes.

**Comment by scottyhq** [comment]
@mikedurand notice the new failing test: https://github.com/snowex-hackweek/website/pull/34/checks?check_run_id=2751220238 due to the notebook containing output cells.

 If you could please do a `Kernel -> Restart kernel and clear all outputs` then re-commit your notebooks we can get this merged.

**Comment by scottyhq** [comment]
looking good @mikedurand ! If you scroll down this page (tab at the top) https://github.com/snowex-hackweek/website/pull/34/files you'll see some typos annotated in case you want to fix them. Otherwise I think we can merge this now that you're familiar with the workflow, and you can add content in a subsequent pull request?

**Review by scottyhq — APPROVED** [review]

**Pull request merged**